### DISCUSSION TITLE
create an android to access to Google sheet

### DISCUSSION THREAD

**Discussion by jeromepou** [community/community#125649] [Programming Help]
### Select Topic Area

Question

### Body

Hi all

I'm beginner. I would like to develop an android. This app should just ask me for which date I want to check the order.
Then for the date select the code must access to a Google sheet, find the date, check which products have been ordered.
Then create a pdf with all the products that have been ordered.

Do you that is possible?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @jeromepou !  

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ebndev** [comment]
Hey @jeromepou, thanks for being a part of the GitHub Community!

I think this guide on [_Implementing Google Sheets API in your Android app_](https://mobileappcircular.com/the-syntax-chronicles-implementing-google-sheets-api-in-your-android-app-8d1bf9fa061a) might be helpful to get you started.

Once a user selects the date, you could send an API request to filter through the Google Sheet based on the date. Maybe if you get that far then you can explore if the API has the ability to export to PDF. If not, it looks like there's a script you can add to accomplish that. [_Generate & send PDFs from Google Sheets_](https://developers.google.com/apps-script/samples/automations/generate-pdfs)

Best of luck! I encourage you to post more discussions if you have questions as you go along 😄
